stackstorm.yml

```diff
@@ -5,6 +5,7 @@
     - mongodb
     - rabbitmq
     - st2repos
+    - postgresql
     - mistral
```

Comment:
Please don't forget to rename `misral` to `st2mistral` as requested in https://github.com/StackStorm/ansible-st2/issues/11

So we'll use the following roles responsible for StackStorm:
- `st2`
- `st2mistral`
- `st2web`
- `st2chatops` in future

for consistency.